charts/opentelemetry-operator/templates/NOTES.txt

```diff
@@ -0,0 +1,6 @@
+{{ $.Chart.Name }} has been installed. Check its status by running:
+  kubectl --namespace {{ .Release.Namespace }} get pods -l "release={{ $.Release.Name }}"
+
+Visit https://github.com/open-telemetry/opentelemetry-operator for instructions on how to create & configure OpenTelemetryCollector and Instrumentation custom resources by using the Operator.
+
+admissionWebhooks flag is mutually exclusive with customAdmissionWebhooks and should not be enabled at the same time. 
```

Comment:
We can go a step further and check if they are both set and fail the install if they are.  The collector chart has an example of this technique.